geomesa-tools/conf/geomesa-env.sh

```diff
@@ -28,6 +28,7 @@ function setvar() {
     export $1=$2
   fi
 }
+GEOMESA_TOOLS_VERSION=%%gmtools.version%%
```

Comment:
I went back and looked at this and the problem I was trying to solve was if a user has multiple versions of the tools we should be able to detect if they forgot to reset the GEOMESA_CONF_DIR before we run anything.